src/DependencyInjection/Configuration.php

```diff
@@ -17,24 +17,16 @@
 
 class Configuration implements ConfigurationInterface
 {
-    private $debug;
-
-    public function __construct($debug)
-    {
-        $this->debug = $debug;
-    }
-
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('knp_disqus');
+        $treeBuilder = new TreeBuilder('knp_disqus');
+        $rootNode = $treeBuilder->getRootNode();
 
         $rootNode
             ->children()
                 ->scalarNode('api_key')->isRequired()->cannotBeEmpty()->end()
                 ->scalarNode('secret_key')->defaultValue('')->end()
                 ->scalarNode('base_url')->defaultValue('')->end()
```

Comment:
Probably `base_url` and `secret_key` can now default to `null` instead of empty quotes.